src/govuk/components/pagination/template.njk

```diff
@@ -0,0 +1,62 @@
+{% set type = params.results.type | default("results") -%}
+
+<nav class="govuk-pagination {{- ' ' + params.classes if params.classes}}" id="pagination-label" aria-label="{{type}} navigation">
+
+  <ul class="govuk-pagination__list">
+    {%- if params.previous %}
+    <li class="govuk-pagination__item govuk-pagination__item--prev">
+      <a class="govuk-pagination__link" href="{{ params.previous.href }}">
+        <span class="govuk-pagination__link-title">
+          <svg class="govuk-pagination__icon" xmlns="http://www.w3.org/2000/svg" height="13" width="17">
+            <path d="m6.5938-0.0078125-6.7266 6.7266 6.7441 6.4062 1.377-1.449-4.1856-3.9768h12.896v-2h-12.984l4.2931-4.293-1.414-1.414z"></path>
+          </svg>{%- if params.previous.text %}{{ params.previous.text }}{% else %}Previous
+          <span class="govuk-visually-hidden">page</span>
```

Comment:
I kinda think we should avoid having default text within the templates? Partly as it makes translation harder, and partly as it just feels a bit unexpected to be able to call the template without specifying the text and have it magically appear.

Could we follow the pattern from the Summary List component here, and have `visuallyHiddenText` as a separate param?

```suggestion
          </svg>{{ params.previous.text }}{% if params.previous.visuallyHiddenText %} 
          <span class="govuk-visually-hidden">{{ params.previous.visuallyHiddenText }}</span>{% endif %}
```

Reply:
Could we allow `params.previous.html` so that people have more control over how the hidden and visible text work together?

For example, if translating into Spanish you'd want to use `<span class="govuk-visually-hidden">página</span> anterior`, which isn't possible with either of the suggested patterns but could be done if you could provide custom HTML.

On the other point: I like the idea of sensible defaults for these kinds of values because it helps keep the components accessible-by-default. I've seen a few accessibility issues in services where someone's used the Design System components and failed to set a necessary parameter. If we can help them out-of-the-box, but provide enough configurability to make choices, I think we should.

Reply:
Really good point about translation order - I think you're right that we should offer html here, as we do with most components.

The issue of sensible defaults is a thorny one! I think I'll leave this thread here as a prompt to have that conversation with the team - feels like something we should agree as a general principle. I do agree though, that the current defaults, whilst handy, are a bit too 'magic'.